client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1315,18 +1315,22 @@
         };
     }
 
-    ContentCreateDirective.$inject = ['api'];
-    function ContentCreateDirective(api) {
+    ContentCreateDirective.$inject = ['api', 'desks'];
+    function ContentCreateDirective(api, desks) {
         var NUM_ITEMS = 5;
         return {
             templateUrl: 'scripts/superdesk-authoring/views/sd-content-create.html',
             link: function(scope) {
+                scope.numItems = NUM_ITEMS;
                 scope.contentTemplates = null;
 
-                var fetchTemplates = function(numItems) {
+                var fetchTemplates = function() {
                     var params = {
-                        max_results: numItems || undefined,
-                        where: {template_type: 'create'}
+                        max_results: NUM_ITEMS || undefined,
```

Comment:
The `undefined` seems redundant now, `NUM_ITEMS` always equals to a truthy value (i.e. 5).